test/loader/loader_config/urLoaderConfigGetInfo.cpp

```diff
@@ -23,30 +23,115 @@ struct urLoaderConfigGetInfoTest
 };
 
 INSTANTIATE_TEST_SUITE_P(
-    , urLoaderConfigGetInfoTest,
+    , urLoaderConfigGetInfoWithParamTest,
     ::testing::Values(UR_LOADER_CONFIG_INFO_AVAILABLE_LAYERS,
                       UR_LOADER_CONFIG_INFO_REFERENCE_COUNT));
 
-TEST_P(urLoaderConfigGetInfoTest, Success) {
+TEST_P(urLoaderConfigGetInfoWithParamTest, Success) {
     ASSERT_SUCCESS(urLoaderConfigGetInfo(loaderConfig, infoType, infoSize,
                                          infoAllocation.data(), nullptr));
 }
 
-TEST_P(urLoaderConfigGetInfoTest, InvalidNullHandleLoaderConfig) {
+TEST_P(urLoaderConfigGetInfoWithParamTest, InvalidNullHandleLoaderConfig) {
     ASSERT_EQ(UR_RESULT_ERROR_INVALID_NULL_HANDLE,
               urLoaderConfigGetInfo(nullptr, infoType, infoSize,
                                     infoAllocation.data(), nullptr));
 }
 
-TEST_P(urLoaderConfigGetInfoTest, InvalidNullPointer) {
+TEST_P(urLoaderConfigGetInfoWithParamTest, InvalidNullPointer) {
+    ASSERT_EQ(
+        UR_RESULT_ERROR_INVALID_NULL_POINTER,
+        urLoaderConfigGetInfo(loaderConfig, infoType, 1, nullptr, nullptr));
+
     ASSERT_EQ(
         UR_RESULT_ERROR_INVALID_NULL_POINTER,
         urLoaderConfigGetInfo(loaderConfig, infoType, 0, nullptr, nullptr));
 }
 
-TEST_P(urLoaderConfigGetInfoTest, InvalidEnumerationInfoType) {
+TEST_P(urLoaderConfigGetInfoWithParamTest, InvalidEnumerationInfoType) {
     ASSERT_EQ(UR_RESULT_ERROR_INVALID_ENUMERATION,
               urLoaderConfigGetInfo(loaderConfig,
                                     UR_LOADER_CONFIG_INFO_FORCE_UINT32, 0,
                                     nullptr, &infoSize));
 }
+
+TEST_P(urLoaderConfigGetInfoWithParamTest, InvalidSize) {
+    ASSERT_EQ(UR_RESULT_ERROR_INVALID_SIZE,
+              urLoaderConfigGetInfo(loaderConfig, infoType, 0,
+                                    infoAllocation.data(), &infoSize));
+
+    ASSERT_EQ(UR_RESULT_ERROR_INVALID_SIZE,
+              urLoaderConfigGetInfo(loaderConfig, infoType, 1,
+                                    infoAllocation.data(), &infoSize));
+}
+
+struct urLoaderConfigGetInfoTest : urLoaderConfigGetInfoWithParamTest {};
```

Comment:
instead of this you can do `using urLoaderConfigGetInfoTest = LoaderConfigTest;` and then you don't need to do the `INSTANTIATE_TEST_SUITE` stuff, you can just define the tests like `TEST_F(urLoaderConfigGetInfoTest, nameOfTest) {`

Reply:
yeah googletest can be a bit awkward sometimes, think I done your suggestion in a later PR as well when I noticed this option. Updated it, cheers!